README.md

```diff
@@ -112,6 +112,26 @@ the `:qmi` [documentation](https://hexdocs.pm/qmi/QMI.Codec.NetworkAccess.html#t
 If this configuration is provided, the modem will use the default configuration
 that provided by the manufacturer.
 
+## Configure roaming
+
+By default the roaming setting that is provided by the modem, however you can
+explicitly enable or disable roaming for a service provider by configuring the
+`:roaming` field of the provider configuration. To allow roaming you can set the
+the field to `:enabled` and to not allow roaming you can set the field to
+`:disabled`.
+
+```elixir
+VintageNet.configure("wwan0", %{
+      type: VintageNetQMI,
+      vintage_net_qmi: %{
+        only_radio_technologies: [:lte, :umts],
+        service_providers: [
+          %{apn: "apn_name", roaming: :enabled}
```

Comment:
```suggestion
          %{apn: "apn_name", roaming_disallowed?: false}
```

It looks like this parameter goes from `:enabled`/`:disabled` to `:allowed`/`:disallowed` to `false'/`true`. That last one is for `:roaming_disallowed`. How about getting rid of all of the intermediate atoms and going straight for the QMI name for the field?

Reply:
Oops the `:enabled`/`:disabled` stuff should be deleted.

I was just trying to make the API make more sense as `roaming_disallowed?: false` seems awkward mentally. To enable roaming we have to set the `roaming_disallowed?` field to `false`, kinda like a double negative. I don't mind doing that as it maps directly to the QMI spec, but having `:roaming` to either be `:allowed` or `:disallowed` seemed a little better from an higher level, API understandability.

Reply:
I have a preference for using `true` and `false` for things that are truly boolean, and this seems like one of those cases given that it's in the QMI spec. My preference is mostly because atoms can be typo'd and it's hard to see typos sometimes and get confused why something doesn't work. If you'd prefer to change to `roaming_allowed?`, that works. I agree on the double negative, and it's also nice that if you don't specify `roaming_allowed?: true`, you get the default behavior of roaming being disallowed. (at least for the modems we're trying, since it's unclear to me what the default actually is)